src/rantanplan/rhymes.py

```diff
@@ -49,7 +49,12 @@ def get_stressed_endings(lines):
         for phonological_group in line["phonological_groups"]:
             # Break groups on last synalepha index position if present
             if "synalepha_index" in phonological_group:
-                synalepha_index = phonological_group["synalepha_index"][-1] + 1
+                has_tilded_vowel = TILDED_VOWELS_RE.search(
+                    phonological_group["syllable"])
+                if has_tilded_vowel:
+                    synalepha_index = has_tilded_vowel.start()
+                else:
+                    synalepha_index = phonological_group["synalepha_index"][-1] + 1
```

Comment:
This chunk could be refactor for both `synalepha_index` and `synaeresis_index`.

Reply:
Done by creating a new function and its tests